packages/ui/src/machines/authenticator/index.ts

```diff
@@ -25,47 +25,58 @@ export function createAuthenticatorMachine() {
       context: {
         user: undefined,
         config: {},
-        services: {},
+        services: { ...defaultServices },
```

Comment:
Curious, can we just pass `defaultServices` here?

Reply:
Oh man, I did it again didn't I :D

Yep, you're right.

```suggestion
        services: defaultServices,
```